### ISSUE TITLE
post_install.py error after fresh pinguino-ide install

### ISSUE THREAD

**Issue by MefhigosetH** [PinguinoIDE/pinguino-ide#29]
Hi @YeisonEng ,

I'm testing a new version of pinguino-NSI-installer an found  that post_install.py doestn't work after a fresh pinguino-ide installation on Windows 7 32-bits.

I've take a screenshot...
![pinguinoide-postinstall-error](https://cloud.githubusercontent.com/assets/527738/5888885/569935da-a3ee-11e4-9a3c-36252ec8f100.png)

What do you think? I've used the pinguino-ide zip package currently hosted on SF.net

Thanks in advanced. Cheers.

**Comment by YeisonEng** [comment]
Hi
I have solved the problem (I hope).
replace the file https://raw.githubusercontent.com/PinguinoIDE/pinguino-ide/master/post_install.py
Until the package is updated.

**Comment by MefhigosetH** [comment]
Hi Yeison,

I try with the new version. This is the expected output for post_install.py file ?
![pinguinoide-postinstall-fixed](https://cloud.githubusercontent.com/assets/527738/5932769/35c448f2-a693-11e4-8704-e96be9dbabba.png)

**Comment by MefhigosetH** [comment]
It's me again.

I think that post_install.py would don't remove all files on source folder. Makefile32.windows and main32.c file would been keep untouch. If not, IDE doesn't compile in 32-bit.

I'm wrong?

**Comment by rblanchot** [comment]
Hi Victor,
There was a bug in the makefile32.windows file.

LKRDIR        =    $(_IDE_P32DIR_)\lkr\$(_IDE_BOARD_)

should be :

LKRDIR        =    $(_IDE_P32DIR_)\lkr

I've updated the packages.

Actually, I changed the lkr directory content and deleted all BOARDS folders with procdefs.ld, elf32pic32mx.x and ISRwrapper.S because these 2 last files are the same for all boards and some boards have the same processors thus the same procdefs.ld
To make it simpler for me to maintain all these files we now only have generic elf32pic32mx.x and ISRwrapper.S for all processors and a proc.ld (32MX270F256B.ld for ex.) for each processor.

**Comment by MefhigosetH** [comment]
Thanks you very much @rblanchot !

I would test again and back with feedback. Cheers.

**Comment by MefhigosetH** [comment]
@YeisonEng , What about the files deleted in the source folder when run post_install.py script. It's ok ? Or there is an issue ?

**Comment by rblanchot** [comment]
I think I've found the problem :
post_install.py remove the whole source directory and then call the PinguinoConfig.check_user_files() defined in qtgui/pinguino_api/pinguino_config.py.
It checks if source directory exists and if not copy it from the install directory to the user directory.
Unfortunately (my fault) Windows paths defined in paths.cfg are the same :

[paths-windows]
install_path = C:\pinguino-11\
user_path = C:\pinguino-11\

The best would be to use the Windows User path :
user_path = ~/pinguino

That's what I did dome time ago but then I got an issue with spaces contains in path.

Questions and remarks : 
1- is it possible to solve the "spaces" issue ?
2- if not, what would be a good path for the user's directory on a Windows System ?
3- it makes no sense to have install and user directories in the same parent directory.
4- any suggestion to enhance this ?
